requirements.sh

```diff
@@ -1,17 +1,22 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-# Determine whether ansible is installed or not.
-info="Please install ansible 1.9.*. Visit http://docs.cibox.tools/en/latest/Requirements for more information."
-is_ansible_installed=false
-ansible --version >/dev/null 2>&1 && { is_ansible_installed=true; }
-if [ "$is_ansible_installed" = false ]; then
-  echo ${info}
+help="Visit http://docs.cibox.tools/en/latest/Requirements for more information."
+ansible=$(which ansible)
+
+if [ -z ${ansible} ]; then
+  echo "Ansible not installed. Further interaction is not possible. ${help}"
   exit 1
```

Comment:
Exit codes must always be different for debugging purposes. This is a basis of shell scripting.